typings/balena-sdk.d.ts

```diff
@@ -584,7 +584,7 @@ declare namespace BalenaSdk {
 		ip?: string;
 		gateway?: string;
 		netmask?: string;
-		version: string;
+		version?: string;
```

Comment:
I think that this always holds a `version`.
See: https://github.com/balena-io/open-balena-api/blob/test-suite-improvements/src/lib/device-config.ts#L93